packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -360,9 +360,21 @@ class FlutterInspectorSettingsDialog extends StatelessWidget {
               'Widgets in these directories will show up in your summary tree.',
               style: theme.subtleTextStyle,
             ),
-            Text(
-              '(e.g. /absolute/path/to/myPackage)',
-              style: theme.subtleTextStyle,
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Text(
+                  '(e.g. /absolute/path/to/myPackage)',
+                  style: theme.subtleTextStyle,
+                ),
+                MoreInfoLink(
+                  url:
+                      'https://docs.flutter.dev/tools/devtools/inspector#package-directories',
+                  gaScreenName: InspectorScreen.id,
+                  gaSelectedItemDescription:
+                      gac.topicDocumentationLink('package_directories'),
```

Comment:
use a constant from the analytics constants `gac`. You can create a new file _inspector_constants.dart with an enum value you can use here. Follow the pattern used by other files in the analytics/constants folder